builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -141,7 +155,18 @@ func resourceAwsApiGatewayMethodUpdate(d *schema.ResourceData, meta interface{})
 	}
 
 	if d.HasChange("request_parameters_in_json") {
-		ops, err := expandApiGatewayMethodParametersJSONOperations(d, "request_parameters_in_json", "requestParameters")
+		ops, err := deprecatedExpandApiGatewayMethodParametersJSONOperations(d, "request_parameters_in_json", "requestParameters")
+		if err != nil {
+			return err
+		}
```

Comment:
I think you're missing `operations = append(operations, ops...)` here which is why tests are failing 😉 
https://travis-ci.org/hashicorp/terraform/builds/150845813#L175